File: 05. Percabangan/If Then Else/If-Then-Else.c
#include <stdio.h>

int main ()
{
    int bilangan = 0;

    scanf ("%i", &bilangan);
    if (bilangan > 0)
    {
        printf ("positif");
    }
    else if (bilangan == 0)
    {
        printf ("nol");
    }
    else if (bilangan < 0)
    {
        printf ("negatif");
    }

    return 0;
}

File: 05. Percabangan/If Then Else/IfThenElse.java
import java.util.Scanner;

public class IfThenElse
{
    public static void main(String [] args)
    {
        Scanner scanner = new Scanner (System.in);

        int bilangan = scanner.nextInt ();

        scanner.close ();

        if (bilangan > 0)
        {
            System.out.println("positif");
        }
        else if (bilangan == 0)
        {
            System.out.println("nol");
        }
        else if (bilangan < 0)
        {
            System.out.println("negatif");
        }
   }    
}
